hello/YearRangePlaylist.py

```diff
@@ -503,8 +503,16 @@ def millennialPlaylist(self):
         playlistId = newPlaylist
         return playlistId
 
-    
-    def showPlaylist(self, passedId):
```

Comment:
This function should not have been deleted, this is what shows the created playlist in the window